content/bash.rst

```diff
@@ -10,14 +10,14 @@ General
 -------
 
 * Shell scripts should be kept at a minimum and **only** for internal scripts.
-  For all other things there is Python.
+  For every other script we should use Python.
 
-* If any, public scripts should use ``/bin/sh``. Bash is not available on all
+* Public shell scripts should use ``/bin/sh``. Bash is not available on all
   Unix systems and ksh or other shells are not compatible with Bash syntax.
 
 * Use ``printf`` instead of ``echo`` when using  ``/bin/sh`` as a shell,
   because ``echo`` behaviour is incompatible in Bash and Korn shells.
-  Eg. use ``printf`` instead of ``echo -n`` and ``printf "blabla\n"`` instead
+  Eg. use ``printf "\n"`` instead of ``echo -n`` and ``printf "blabla\n"`` instead
```

Comment:
Not at all, quite the opposite as 'echo -n' removes the trailing newline. 
My mistake - I tested what I wanted to rewrite then, but I must have modified it in the end without a last test. Will be more careful, sorry!!